src/types.ts

```diff
@@ -39,16 +45,17 @@ export interface Column<TRow, TSummaryRow = unknown> {
   readonly sortable?: Maybe<boolean>;
   /** Sets the column sort order to be descending instead of ascending the first time the column is sorted */
   readonly sortDescendingFirst?: Maybe<boolean>;
-  /** Editor to be rendered when cell of column is being edited. If set, then the column is automatically set to be editable */
-  readonly editor?: Maybe<(props: EditorProps<TRow, TSummaryRow>) => ReactNode>;
   readonly editorOptions?: Maybe<{
-    /** @default false */
-    readonly renderFormatter?: Maybe<boolean>;
+    /**
+     * Render the cell content in addition to the edit cell.
+     * Enable this option when the editor is rendered outside the grid, like a modal for example.
+     * By default, the cell content is not rendered when the edit cell is open.
+     * @default false
+     */
+    readonly displayCell?: Maybe<boolean>;
```

Comment:
`renderFormatter` would have been a bit confusing I think, so I renamed it to `displayCell`.
Maybe `displayCellContent` is better though, wdyt?

Reply:
yeah, let's rename it to `displayCellContent`

Reply:
Done